src/designer/designer_packages/elsa-workflows-designer/src/modules/workflow-definitions/plugins/workflow-definitions-plugin.tsx

```diff
@@ -223,8 +223,19 @@ export class WorkflowDefinitionsPlugin implements Plugin {
     });
 
     await this.saveWorkflowDefinition(definition, true)
-      .then(async () => {
+      .then(async (response) => {
         NotificationService.updateNotification(notification, {title: 'Workflow published', text: 'Published!'})
+
+        setTimeout(function() {
+          if(response.consumingWorkflowsBeingUpdated.length > 0)
+          NotificationService.createNotification({
+            title: 'Consuming Workflows',
+            id: uuid(),
+            text: 'Following consuming workflows are being published to update the references of this composite activity:\n\n' + response.consumingWorkflowsBeingUpdated.join('\n'),
```

Comment:
Proper English would be to start the sentence with: `The` ('The following consuming ya da ya da').